src/main/java/com/aws/greengrass/mqttclient/MqttClient.java

```diff
@@ -432,6 +447,15 @@ public synchronized void unsubscribe(UnsubscribeRequest request)
             return future;
         }
 
+        try {
+            isValidPublishRequest(request);
+        } catch (SpoolerStoreException e) {
+            logger.atDebug().kv("topic", request.getTopic())
```

Comment:
provide a clear log message such as `invalid publish request: {}` and then interpolate the error message there.

Reply:
Probably should log this as an error.